### PR TITLE
New about

### PR DESCRIPTION
**Pull request by yesroh**
Making some changes, forgiveness for the English. It is not my mother tongue. I am open to advice and suggestions #96 #97

### CODE REVIEW

File: docs/ABOUT.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+PHP is also a language that has broad community support (since 1995) with a big user base, and a clear and simple documentation.
+
+You can learn more about PHP [here](http://php.net/manual/en/intro-whatis.php)
+
+
+
+Originally ,[4] the PHP reference implementation is now produced by The PHP Group.[5]
+
+Was originally created by Rasmus Lerdorf in 1994 to ad
```
Comment (kytrinyx) [inline docs/ABOUT.md]: I think this is a stray line—we can delete it.

File: docs/ABOUT.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# PHP
+
+PHP is a development language intended for the web that has become popular because it is easy to learn, over time it has become a modern language and so won its space in the roll of the most respected programming languages. It was originally created by Rasmus Lerdorf in 1994 to manager your personal blog. To get started with PHP is very simple! just add some PHP -tags to an already existing HTML file and voila!
```
Comment (kytrinyx) [inline docs/ABOUT.md]: Let's change the first comma into a period—both of those read as full sentences, and shorter sentences tend to be easier to read.

> to manager your personal blog.

How about: _to make it easier to manage a personal website._

> To get started with PHP is very simple!

How about: _Getting started with PHP is very simple!_

> just add some PHP -tags

Capitalize `Just`, and remove the space before `-tags`.

File: docs/ABOUT.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
```
Comment (kytrinyx) [inline docs/ABOUT.md]: Let's also mention that it has grown and it's no longer just for personal home pages (e.g. Facebook, Etsy, WordPress).

File: docs/ABOUT.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+PHP is also a language that has broad community support (since 1995) with a big user base, and a clear and simple documentation.
+
+You can learn more about PHP [here](http://php.net/manual/en/intro-whatis.php)
+
+
+
+Originally ,[4] the PHP reference implementation is now produced by The PHP Group.[5]
```
Comment (kytrinyx) [inline docs/ABOUT.md]: What are the 4 and 5 about?

**Comment by kytrinyx** [comment]
Thanks @yesroh! I like the friendly, conversational tone in this.

**Comment by petemcfarlane** [comment]
closing, updated in #100

**Pull request closed without merging**